src/response.rs

```diff
@@ -522,7 +519,7 @@ impl<'a, 'b> Response<'a, 'b> {
         let mime = path
             .extension()
             .and_then(|ext| to_mime(&ext.to_string_lossy()))
-            .unwrap_or(Mime(TopLevel::Application, SubLevel::Ext("octet-stream".into()), vec![]));
+            .unwrap_or_else(|| Mime(TopLevel::Application, SubLevel::Ext("octet-stream".into()), vec![]));
```

Comment:
Oh, my! My brain must have had a meltdown. :persevere: I thought, for some reason, that the change was from `unwrap_or_else` to `unwrap_or`. Ignore that comment :sweat_smile: